roles/siege-run/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Coppied from mycourses load-generator.yml example
+- name: Configure Siege
+  template:
+    src: "files/siege.conf"
+    dest: "/.sigerc"
+    seuser: system_u
+
+- name: Push hosts file
+  template:
+    src: "files/hosts.conf"
+    dest: "/usr/local/etc/urls.txt"
```

Comment:
Just curious, any particular reason for this `dest` path?

Reply:
No reason in particular, just what I saw in an example